### PR TITLE
run pact services synchronization on all chains concurrently

### PR DESCRIPTION
**Pull request by larskuhtz**

### CODE REVIEW

File: src/Chainweb/Chainweb.hs [hunk 1]
```diff
@@ -455,7 +455,7 @@ withChainwebInternal conf logger peer rocksDb dbDir nodeid resetDb inner = do
 
     synchronizePactDb cs cutDb = do
         currentCut <- _cut cutDb
-        mapM_ syncOne $ mergeCutResources $ _cutMap currentCut
+        mapConcurrently_ syncOne $ mergeCutResources $ _cutMap currentCut
```
Comment (fosskers) [inline src/Chainweb/Chainweb.hs]: This will use all available threads. Is that desired?

Comment (larskuhtz) [reply src/Chainweb/Chainweb.hs]: For 20 chains during initialization, yes. It should be almost a no-op in most cases (we are investigating currently why it isn't currently). Also, if at all it takes longer it should be IO bound.

Comment (larskuhtz) [reply src/Chainweb/Chainweb.hs]: But once we go to, say >100, chains, we should use a scheduler.

Comment (larskuhtz) [reply src/Chainweb/Chainweb.hs]: @fosskers if you are familiar with the scheduler package and know how to fix this quickly, please do it.

**Pull request merged**